tracing-subscriber/src/fmt/mod.rs

```diff
@@ -642,18 +642,33 @@ where
     /// - `FmtSpan::NONE`: No events will be synthesized when spans are
     ///    created, entered, exited, or closed. Data from spans will still be
     ///    included as the context for formatted events. This is the default.
-    /// - `FmtSpan::ACTIVE`: Events will be synthesized when spans are entered
-    ///    or exited.
+    /// - `FmtSpan::NEW`: Events will be synthesized when spans are created.
+    /// - `FmtSpan::ENTER`: Events will be synthesized when spans are entered.
+    /// - `FmtSpan::EXIT`: Events will be synthesized when spans are exited.
     /// - `FmtSpan::CLOSE`: An event will be synthesized when a span closes. If
     ///    [timestamps are enabled][time] for this formatter, the generated
     ///    event will contain fields with the span's _busy time_ (the total
     ///    time for which it was entered) and _idle time_ (the total time that
     ///    the span existed but was not entered).
+    /// - `FmtSpan::ACTIVE`: Events will be synthesized when spans are entered
+    ///    or exited.
```

Comment:
```suggestion
    /// - `FmtSpan::ACTIVE`: An event will be synthesized when spans are entered
    ///    or exited.
```

Also, question for @hawkw: should we deprecate this variant and pushing folks to use `FmtSpan::Enter | FmtSpan::Exit` instead?

Reply:
(These changes have been merged, but unresolving to make this discussion more visible.)

Reply:
IMHO `ACTIVE` is not the most intuitive name and it would be more natural if you just combined them explicitly. That would make things more "self-documenting" with the code being more obvious and longer descriptions of the meaning of flags less necessary.